src/layouts/MainLayout.astro

```diff
@@ -8,9 +9,36 @@ const {title} = Astro.props
 ---
 
 <Layout title={title}>
-  <body class="bg-zinc-100">
-    <div class="w-[500px] m-auto">
-      <slot />
-    </div>
-  </body>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+      <meta name="viewport" content="width=device-width" />
+      <meta name="generator" content={Astro.generator} />
+      <title>{title ? `${title} | ${copy.defaultPageTitle}` : copy.defaultPageTitle}</title>
+      <style>
+        body {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          height: 100vh;
+          font-family: Arial, sans-serif;
+          background-color: #29222b;
+          background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='793' height='95.2' viewBox='0 0 1000 120'%3E%3Cg fill='none' stroke='%231C1917' stroke-width='25' %3E%3Cpath d='M-500 75c0 0 125-30 250-30S0 75 0 75s125 30 250 30s250-30 250-30s125-30 250-30s250 30 250 30s125 30 250 30s250-30 250-30'/%3E%3Cpath d='M-500 45c0 0 125-30 250-30S0 45 0 45s125 30 250 30s250-30 250-30s125-30 250-30s250 30 250 30s125 30 250 30s250-30 250-30'/%3E%3Cpath d='M-500 105c0 0 125-30 250-30S0 105 0 105s125 30 250 30s250-30 250-30s125-30 250-30s250 30 250 30s125 30 250 30s250-30 250-30'/%3E%3Cpath d='M-500 15c0 0 125-30 250-30S0 15 0 15s125 30 250 30s250-30 250-30s125-30 250-30s250 30 250 30s125 30 250 30s250-30 250-30'/%3E%3Cpath d='M-500-15c0 0 125-30 250-30S0-15 0-15s125 30 250 30s250-30 250-30s125-30 250-30s250 30 250 30s125 30 250 30s250-30 250-30'/%3E%3Cpath d='M-500 135c0 0 125-30 250-30S0 135 0 135s125 30 250 30s250-30 250-30s125-30 250-30s250 30 250 30s125 30 250 30s250-30 250-30'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
Here for svgs I'd prefer it if we added them as assets in our public/css folder and pulled them in using astro so they're 1) easy to minify 2) easy to update and track updates 3) have transparency on what the content of the svg is https://github.com/astoria-tech/meetup/blob/main/public/css/wavy-background.svg

example of how to do this in astro here:

https://github.com/astoria-tech/meetup/blob/main/src/pages/links.astro#L9

Reply:
- Use the file instead of SVG
- Pass the background as a boolean e.g. `wavy-bg: true`
- Make Tailwind classes for reusable colors

Reply:
Resolved via [feat: useWavyBackground](https://github.com/astoria-tech/meetup/commit/00d9f9c230cc7ba178eb276ab9c89c5ea9171546)

Tailwind color classes to be added by @ninjaofawesome